docs/ref/utils.txt

```diff
@@ -31,6 +31,9 @@ cache keys to prevent delivery of wrong content.
 For example, :doc:`internationalization </topics/i18n/index>` middleware would
 need to distinguish caches by the ``Accept-language`` header.
 
+patch_cache_control()
+---------------------
```

Comment:
I think all of the headers you have added should be surrounded in double backticks as noted above.